### ISSUE TITLE
Links not underlined in agenda packet tab

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2099]
**Source**

Fairfax

**Describe the bug**

The links in the agenda packet tab aren't underlined. It appears this is just for those that are in an `li`.

I added this custom CSS to fix it on the Fairfax website, but we should add to the main CSS file:

```
#tab-agenda-packet li a, #tab-agenda-packet li a:visited {
	text-decoration: underline;
}

```
**To Reproduce**
1. Go to [['...'](https://www.townoffairfax.org/meetings/town-council-meeting-august-9-2022/)](https://www.townoffairfax.org/meetings/town-council-meeting-august-9-2022/)
2. Click on 'Agend packet' tab
3. Inspect code to remove custom CSS

**Expected behavior**

Links in main content aren't should be underlined.

**Screenshots**

N/A

**Desktop (please complete the following information):**

N/A

**Smartphone (please complete the following information):**

N/A

**Additional context**

N/A

**Comment by curtismchale** [comment]
@lukefretwell I took a look at the site and I don't see any custom CSS rules.

<img width="617" alt="CleanShot 2022-09-06 at 13 56 31@2x" src="https://user-images.githubusercontent.com/74561/188737062-5d6948d9-0265-4368-bf22-db154f987d78.png">

I also checked Beta and it appears that the links are working there for agenda packets without further CSS work.

<img width="608" alt="CleanShot 2022-09-06 at 14 02 46@2x" src="https://user-images.githubusercontent.com/74561/188737607-877b97d1-e8cb-41c8-8ebf-2e152726a996.png">

You won't be able to switch to the agenda packet tab until I deploy the latest build process update in a few minutes.

**Comment by lukefretwell** [comment]
@curtismchale see the 'Appoint' link (at the bottom of the screenshot) which is not underlined. I can't figure out why it's not underlined, but looks like it's for a list item.

**Comment by curtismchale** [comment]
@lukefretwell yup I see it. There doesn't appear to be a blank rule for links to make them underlined. The rule that applies to the rest of the content matches on `p a`. I'll look at a better generic rule...without affecting everything else.

**Comment by curtismchale** [comment]
@lukefretwell new rules going up with the new build to Beta shortly.
